tests/expressions_window/tests.py

```diff
@@ -637,15 +637,18 @@ def test_distinct_window_function(self):
 
     def test_fail_update(self):
         """Window expressions can't be used in an UPDATE statement."""
-        msg = 'Window expressions are not allowed in this query'
+        msg = 'Window expressions are not allowed in this query ("Max(Col(expressions_window_employee, ' \
+              'expressions_window.Employee.salary)) OVER (PARTITION BY Col(expressions_window_employee, ' \
+              'expressions_window.Employee.department))" on expressions_window.Employee.salary).'
```

Comment:
Use parentheses instead of backslash escaping

```python
msg = (
    'Window expressions are not allowed in this query ("Max(Col(expressions_window_employee, '
    'expressions_window.Employee.salary)) OVER (PARTITION BY Col(expressions_window_employee, '
    'expressions_window.Employee.department))" on expressions_window.Employee.salary).'
)